.gitbook.yml

```diff
@@ -0,0 +1 @@
+root: ./lilypad/
```

Comment:
😍

Reply:
Sadly it didn't work and we suspect it might need a complete rebase to fix it 

https://docs.gitbook.com/integrations/git-sync/monorepos